vehicle/as/README.md

```diff
@@ -1,15 +1,82 @@
+  | `/pacmod/parsed_tx/steer_rpt`       | pacmod3_msgs::msg::SystemRptFloat | current steering wheel angle                                            |
+  | `/pacmod/parsed_tx/wheel_speed_rpt` | pacmod3_msgs::msg::WheelSpeedRpt  | current wheel speed                                                     |
+  | `/pacmod/parsed_tx/accel_rpt`       | pacmod3_msgs::msg::SystemRptFloat | current accel pedal                                                     |
+  | `/pacmod/parsed_tx/brake_rpt`       | pacmod3_msgs::msg::SystemRptFloat | current brake pedal                                                     |
+  | `/pacmod/parsed_tx/shift_rpt`       | pacmod3_msgs::msg::SystemRptInt   | current gear status                                                     |
+  | `/pacmod/parsed_tx/turn_rpt`        | pacmod3_msgs::msg::SystemRptInt   | current turn indicators status                                          |
+  | `/pacmod/parsed_tx/global_rpt`      | pacmod3_msgs::msg::GlobalRpt      | current status of other parameters (e.g. override_active, can_time_out) |
+
+#### Output topics
+
+- To Pacmod
+
+  | Name                         | Type                              | Description                                           |
+  | ---------------------------- | --------------------------------- | ----------------------------------------------------- |
+  | `pacmod/as_rx/accel_cmd`     | pacmod3_msgs::msg::SystemCmdFloat | accel pedal command                                   |
+  | `pacmod/as_rx/brake_cmd`     | pacmod3_msgs::msg::SystemCmdFloat | brake pedal command                                   |
+  | `pacmod/as_rx/steer_cmd`     | pacmod3_msgs::msg::SystemCmdFloat | steering wheel angle and angular velocity command     |
+  | `pacmod/as_rx/shift_cmd`     | pacmod3_msgs::msg::SystemCmdInt   | gear command                                          |
+  | `pacmod/as_rx/raw_steer_cmd` | pacmod3_msgs::msg::SteerSystemCmd | raw steering wheel angle and angular velocity command |
+
+- To Autoware
+
+  | Name                               | Type                                                   | Description                                          |
+  | ---------------------------------- | ------------------------------------------------------ | ---------------------------------------------------- |
+  | `/vehicle/status/control_mode`     | autoware_auto_vehicle_mgs::msg::ControlModeReport      | control mode                                         |
+  | `/vehicle/status/twist`            | autoware_auto_vehicle_mgs::msg::VelocityReport         | velocity                                             |
+  | `/vehicle/status/steering`         | autoware_auto_vehicle_mgs::msg::SteeringReport         | steering wheel angle                                 |
+  | `/vehicle/status/gear_cmd`         | autoware_auto_vehicle_mgs::msg::GearReport             | gear status                                          |
```

Comment:
Could you change the topic name to "/vehicle/status/shift" ?
(Change the code as well)

Reply:
Also, control/gear_cmd -> control/shift_cmd

Reply:
In the P/C sync mtg (currently conducted), it was decided that we use not shift but gear for topic names. Is it ok?

Reply:
OK!!
 (/vehicle/status/gear_cmd  ? /vehicle/status/gear? )

Reply:
Thanks, this one. `/vehicle/status/gear_cmd`

Reply:
Thn, could you create the PR to change shift -> gear of other packages?

Reply:
Alright. Sugahara-san is working on some packages and will replace those message name, and I will work on other packages.

Reply:
Ah but now I found that gear_cmd for GearReport is a bit weird...

Reply:
Please going with `/vehicle/status/gear`!

Reply:
We decided to use formats like `/vehicle/status/gear_status`